README.dev.md

```diff
@@ -155,4 +155,10 @@ make doctest
    - `minor`: when functionality was added in a backward compatible manner
    - `path`: when backward compatible bug fixes were made
 4. Merge the release branch into `main`.
-5. Go to https://github.com/EIT-ALIVE/eitprocessing/releases and draft a new release; create a new tag for the release, generate release notes automatically and adjust them, and finally publish the release as latest. This will trigger [a GitHub action](https://github.com/EIT-ALIVE/eitprocessing/actions/workflows/release.yml) that will take care of publishing the package on PyPi.
+5. On the [Releases page](https://github.com/EIT-ALIVE/eitprocessing/releases):
+   1. Click "Draft a new release"
+   2. By convention, use `v<version number>` as both the release title and as a tag for the release.
```

Comment:
Instead of adding tag numbers here, couldn't this be done by bumpversion by adding tag=True to the config? Otherwise, you would have to look up what the version number is yourself, I guess in one of the n places it is coded.

Reply:
Looks like that could work. We could even set `commit = True` as well, which creates a useful automatic commit message.
According to the [documentation](https://pypi.org/project/bumpversion/#configuration) it would require a push with `--tags` flag, which I am bound to forget to do. If we do add it to bumpversion, I would still leave this comment for people like me (and if it's prefilled that's fine).
You'd still need to manually set the title of the release (although then you could immediately see it in the tag and don't have to look it up in the code).

Reply:
Ah, yes, pushing tags is something that's easy to forget. When I read the documentation again, I also think it does not work in this order. It would tag the release branch, not the main branch. I don't think that's what we want. There are some solutions:
- leave as is and create the tag inside GitHub (probably the easiest)
- adapt the documentation to have bumpversion bump and tag after merging to main, and pushing the tag to GitHub
- make a release branch; when making a release, merge main to release, bumpversion, and merge back to main; I think this is used quite often, but it is a bit more complicated.

Let's just go with the first one for now, and not have bumpversion create tags.